src/logic/map_objects/tribes/productionsite.cc

```diff
@@ -629,30 +629,30 @@ void ProductionSite::try_replace_worker(const Game* game,
                                         DescriptionIndex worker_index,
                                         WorkingPosition* wp) {
 	// Search replacement worker in the same building
-	for (auto wp_repl = working_positions_.begin(); wp_repl != working_positions_.end(); ++wp_repl) {
-		if (!wp_repl->worker.is_set()) {
+	for (auto& wp_repl : working_positions_) {
```

Comment:
these changes seem to be unrelated to the purpose of this Branch what is the purpose?

Reply:
MSVC refused to compile line 655 (`wp_repl.base()`). Therefore I had to modify the code to use references instead of iterators.
We will have such compile differences sometimes in the future, now that MSVC compiles every version ;)

Reply:
ok thanks for explanation